tasks/first_server.yml

```diff
@@ -6,7 +6,7 @@
     path: /etc/rancher/rke2
     owner: root
     group: root
-    mode: 0755
+    mode: 0750
```

Comment:
Modifying file and directory permissions to differ from those in the upstream RKE2 installation is not within the scope of this Ansible Role.